Custom.pm

```diff
@@ -51,12 +53,16 @@ sub dirsFor {
 	
 	if ($dir eq 'updates') {
 
-        mkdir UPDATE_DIR unless -d UPDATE_DIR;
-        	
+		mkdir UPDATE_DIR unless -d UPDATE_DIR;
+
 		@dirs = (UPDATE_DIR);
 	}
 	else {
 		@dirs = $class->SUPER::dirsFor($dir);
+
+		if ($dir eq "Plugins") {
+			push @dirs, catdir( Slim::Utils::Prefs::preferences('server')->get('cachedir'), 'Plugins' );
+		}
```

Comment:
Sorry about that. I thought I had verified my change before submitting the pull request. But you're right. We need to update @INC there (though I think this should be dealt with in the plugin manager - but that's another story :-)).

In other places we use unshift rather than push, to put it in front of the search list.